apis/python/tests/conftest.py

```diff
@@ -10,31 +10,31 @@
 
 
 @pytest.fixture
-def h5ad_path(request):
+def conftest_h5ad_path(request):
```

Comment:
> Defining fixtures in conftest.py is a relatively core pytest pattern ([docs](https://docs.pytest.org/en/8.2.x/reference/fixtures.html#conftest-py-sharing-fixtures-across-multiple-files)), and the more unique/descriptive names in https://github.com/single-cell-data/TileDB-SOMA/pull/2529 should make it easy for people to make the connection, in the worst case

> IntelliJ

> I'd rather strike a balance where this pytest "feature" is reasonably discoverable, without scare-{naming,commenting} them at every use-site

Here is where we differ

I'm disinteresting in biasing toward individual toolchains (e.g. IntelliJ). Also since we are multi-lingual it's really crucial that we bias toward less Python expertise. Whatever may seem "standard" to highly evolved Python devs such as yourself -- whom I admire & envy -- there are lots of folks "dipping in" to Python and we need to lower the barrier to entry. I don't find these to be "scare" names -- rather, this is simple inclusivity.